creator-node/src/dbManager.js

```diff
@@ -468,6 +469,35 @@ class DBManager {
     return DBManager._getLegacyStoragePathsAndCids(cursor, batchSize, true)
   }
 
+  static async getCustomStoragePathsRecords(cursor, batchSize) {
+    const queryResult = await models.File.findAll({
+      attributes: [
+        'storagePath',
+        'multihash',
+        'dirMultihash',
+        'fileName',
+        'trackBlockchainId',
+        'cnodeUserUUID'
+      ],
+      where: {
```

Comment:
we should confirm that the 2 db queries for legacy vs custom will not duplicate each other - i.e. any given file should only show up in at most 1 of these queries

Reply:
I did this locally just now